ifrs17-template/Initialization/InitSystemorphUtils.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+    "metadata": {
+        "authors": [],
+        "id": "6vRP2T2AiEq5rmfljOKoSQ",
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "<p style=\"font-weight:bold;\"> <span style=\"font-size: 36px\"> In Memory Initialization of unconfigured DataSource </span> </p>",
+                "\n",
+                "\n<a id='init-to-memory'></a>"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "#!eval-notebook \"InitSystemorphRefDataToMemory\""
```

Comment:
Why should RefData imported in the Utils NB?
Move it to the Init NB

Reply:
done